docs/_getting-started/index.md

```diff
@@ -0,0 +1,127 @@
+
+Until then grab a Windows installer from our [downloads](https://geth.ethereum.org/downloads) page.
+
+### Download standalone binary
+
+We distribute all our stable releases and development builds as standalone binaries. These are useful for scenarios where you'd like to: a) install a specific version of our code (e.g., for reproducible environments); b) install on machines without internet access (e.g., air gapped computers); or c) do not like automatic updates and would rather manually install software.
+
+We create the following standalone binaries:
+
+-   32bit, 64bit, ARMv5, ARMv6, ARMv7 and ARM64 archives (`.tar.gz`) on Linux
+-   64bit archives (`.tar.gz`) on macOS
+-   32bit and 64bit archives (`.zip`) and installers (`.exe`) on Windows
+
+For all binaries we provide two options, one containing only Geth, and another containing Geth along with all the developer tools from our repository (`abigen`, `bootnode`, `disasm`, `evm`, `rlpdump`). Read our [`README`](https://github.com/ethereum/go-ethereum#executables) for more information about these executables.
+
+To download these binaries, head to the [Go Ethereum Downloads](https://geth.ethereum.org/downloads) page.
+
+### Run inside docker container
+
+We maintain a Docker image with recent snapshot builds from our `develop` branch on DockerHub. In addition to the container based on Ubuntu (158 MB), there is a smaller image using Alpine Linux (35 MB). To use the alpine [tag](https://hub.docker.com/r/ethereum/client-go/tags), replace `ethereum/client-go` with `ethereum/client-go:alpine` in the examples below.
+
+To pull the image and start a node, run these commands:
+
+```shell
+docker pull ethereum/client-go
+docker run -it -p 30303:30303 ethereum/client-go
+```
+
+_[Read this guide](/install-and-build/Installing-Geth#run-inside-docker-container) for further Docker options._
+
+## Starting a node
+
+### Create an account
+
+Before starting Geth you first need to create an account that represents a key pair. Use the following command to create a new account and set a password for that account:
+
+```shell
+geth account new
+```
+
+_[Read this guide](/interface/Managing-your-accounts) for more details on importing existing Ethereum accounts and other uses of the `account` command._
+
+### Sync modes
+
+Running Geth starts an Ethereum node that can join any existing network, or create a new one. You can start Geth in one of three different sync modes using the `--syncmode "{mode}"` argument that determines what sort of node it is in the network.
+
+These are:
+
+-   **Full**: Downloads all block headers, block data, and validates all transactions
+-   **Fast** (Default): Downloads block headers and block data of the most recent transactions (1024) and validates them.
```

Comment:
Nope. Fast downloads all blocks ( headers + transactions + receipts), verifies all headers, anda also downloads the state and verifies against the headers.

Reply:
Thanks @holiman here's my change based on your comments. I think I'd like to add a clarification here of what you mean exactly by "state". Bearing in mind this is a getting started guide, what does it mean to you?

```suggestion
-   **Fast** (Default): Downloads all blocks (including headers, transactions and receipts), verifies all headers, and downloads the state and verifies it against the headers.
```

Reply:
Or maybe the state question is answered below…?

Reply:
`state` means 
- All contracts, nonces, balances, storage entries

The state is stored in a merkle trie, and the state at any block can be uniquely identified by the root hash of the tree at that point in time.

So block-by-block generates the state by executing transactions sequentially. Whereas fast-sync downloads the state from peers, and then just validates that the trie root hash indeed corresponds to what the header says that the root-hash should be